core/src/io/anuke/mindustry/world/blocks/power/PowerDiode.java

```diff
@@ -0,0 +1,85 @@
+package io.anuke.mindustry.world.blocks.power;
+
+import io.anuke.arc.Core;
+import io.anuke.arc.graphics.g2d.Draw;
+import io.anuke.arc.graphics.g2d.TextureRegion;
+import io.anuke.arc.math.Mathf;
+import io.anuke.mindustry.world.Block;
+import io.anuke.mindustry.world.Tile;
+
+import static io.anuke.mindustry.Vars.world;
+
+public class PowerDiode extends Block{
+
```

Comment:
This extra indent is unnecessary.

Reply:
I don't see which indentation is out of place 🤔

Reply:
i don't see which indentation is out of place 🤔

Reply:
Sorry, I meant blank line, not indent.